src/components/view-switcher/ViewSwitcher.jsx

```diff
@@ -0,0 +1,40 @@
+import { useState } from 'react'
+import ToggleButton from '@mui/material/ToggleButton'
+
+import FormatListBulletedIcon from '@mui/icons-material/FormatListBulleted'
+import GridViewIcon from '@mui/icons-material/GridView'
+
+import { cardsViews } from '~/constants'
+import { styles } from './ViewSwitcher.styles'
+
+const ViewSwitcher = ({ setOffersView }) => {
+  const [cardsView, setCardsView ] = useState(cardsViews.inline)
+
+  const changeCardsView = (_event,view) => {
+    setOffersView(view)
```

Comment:
do you need second state where you set the same value?

Reply:
useState inside is for setting value of current view to see active button
you mean it would be better if state of current view will come
from props?

Reply:
if you have a state that controls the view setting outside you don't need it here

Reply:
i have deleted useState and change useing local state to state from props

Reply:
@MickhaelSh , would it be a good idea if i will create separate file 'data.js' that will be containing array of objects with field needed for every "ToggleButton" props and then just map them or it will be redundant?